### ISSUE TITLE
error in trigger event line 26

### ISSUE THREAD

**Issue by filipxcs** [BrunoMarc123/BR-Menu#1]
when i try to send table in params i get error when the trigger event have table params only when i send table

**Comment by BrunoMarc123** [comment]
Hello,
Do you mean when trying to send table parameter? What happend than?

**Comment by filipxcs** [comment]
yeah when i send a table i get this errors https://prnt.sc/113kevr the data before i click its perfect https://prnt.sc/113kg25 
https://prnt.sc/113kf5l https://prnt.sc/113kfge i thing this can helps

**Comment by BrunoMarc123** [comment]
I know why, i did that the parameter will be as data-param on html so its not store it inside like needed,
I will work on fix tomorrow and then I will post it.
Thanks for your time

**Comment by filipxcs** [comment]
no problem mate and very nice work btw

**Comment by filipxcs** [comment]
any news?

**Comment by JoeSzymkowiczFiveM** [comment]
I fixed this in my fork. Too bad developer not accepting PRs. My repo is now private.

**Comment by filipxcs** [comment]
@JoeSzymkowiczFiveM maybe can you share the fix ?
